### PR TITLE
DF 2165 - OS Training Data Batch

### PR DESCRIPTION
**Pull request by emina-str**
- first draft

### CODE REVIEW

File: playbooks/deploy-tasks-training.yml [hunk 1]
```diff
@@ -0,0 +1,111 @@
+---
```
Comment (birom) [inline playbooks/deploy-tasks-training.yml]: We should not have environment specific playbooks. The batch jobs can be enabled and disabled on different environment with the when parts eg.:``when: jobroom.jobadservice.prisme_import_task.enabled`` parts.

Comment (emina-str) [reply playbooks/deploy-tasks-training.yml]: aha. ok. will delete it and add the property.

File: playbooks/deploy-training-delete-data-batch.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
```
Comment (birom) [inline playbooks/deploy-training-delete-data-batch.yml]: Why do we need this? Is the application not deployed by the ``deploy-task`` playbook to the SCDF?

File: playbooks/deploy-training-initialize-data-batch.yml [hunk 3]
```diff
@@ -0,0 +1,27 @@
+---
+# FIXME TODO
+- name: Deployment of the Alv Initialize Data Batch (Training)
```
Comment (birom) [inline playbooks/deploy-training-initialize-data-batch.yml]: Why do we need this? Is the application not deployed by the deploy-task playbook to the SCDF?

File: playbooks/deploy-training-delete-data-batch.yml (hunk 2 above)
Comment (emina-str) [reply playbooks/deploy-training-delete-data-batch.yml]: I wasn't sure do we need it or not. when it's only about the batch-job. I'll delete it.
Thanks.

File: playbooks/deploy-training-initialize-data-batch.yml (hunk 3 above)
Comment (emina-str) [reply playbooks/deploy-training-initialize-data-batch.yml]: Will delete it.

File: playbooks/deploy-tasks-training.yml (hunk 1 above)
Comment (emina-str) [reply playbooks/deploy-tasks-training.yml]: @birom  do I need to add the app-name with this properties to all environments, or is it ok if I add it only to training, where it should be deployed/enabled?

File: playbooks/deploy-tasks.yml [hunk 4]
```diff
@@ -45,6 +45,7 @@
       cron_hour: "4"
       cron_minute: "30"
       additional_execution_arguments: "start-date=`date +\\%FT\\%T`"
+      when: jobroom.candidateservice.candidate_import_task.enabled
```
Comment (emina-str) [inline playbooks/deploy-tasks.yml]: @birom  could I maybe write here just `!jobroom.onlineformservice.mock_data.enabled` ?

Comment (birom) [reply playbooks/deploy-tasks.yml]: Yes, good Idea. Please try it out.

**Pull request merged**